src/app/design/frontend/base/default/layout/shopgate/cloudapi/customer.xml

```diff
@@ -0,0 +1,13 @@
+<layout version="0.1.0">
```

Comment:
The layout files are xml files, please add
<?xml version="1.0"?>
as first line

Reply:
done